oc-chef-pedant/spec/api/stats_spec.rb

```diff
@@ -31,15 +31,39 @@
     "erlang_vm_statistics_context_switches" => "COUNTER",
     "erlang_vm_memory_system_bytes_total" => "GAUGE",
     "erlang_vm_memory_processes_bytes_total" => "GAUGE",
-    "erlang_vm_ets_tables" => "GAUGE",
-    "erlang_vm_dets_tables" => "GAUGE",
+    "erlang_vm_memory_ets_tables" => "GAUGE",
+    "erlang_vm_memory_dets_tables" => "GAUGE",
     "erlang_vm_memory_bytes_total" => "GAUGE",
     "erlang_vm_memory_atom_bytes_total" => "GAUGE",
     "erchef_pooler_queued_requestors_max" => "GAUGE",
     "erchef_pooler_queued_requestors" => "GAUGE",
     "erchef_pooler_members_max" => "GAUGE",
     "erchef_pooler_members_free" => "GAUGE",
-    "erchef_pooler_members_in_use" => "GAUGE"
+    "erchef_pooler_members_in_use" => "GAUGE",
+    "erlang_vm_allocators" => "GAUGE",
+    "erlang_vm_atom_count" => "GAUGE",
+    "erlang_vm_atom_limit" => "GAUGE",
+    "erlang_vm_dirty_cpu_schedulers" => "GAUGE",
+    "erlang_vm_dirty_cpu_schedulers_online" => "GAUGE",
+    "erlang_vm_dirty_io_schedulers" => "GAUGE",
+    "erlang_vm_msacc_aux_seconds_total" => "COUNTER",
+    "erlang_vm_msacc_check_io_seconds_total" => "COUNTER",
+    "erlang_vm_msacc_emulator_seconds_total" => "COUNTER",
+    "erlang_vm_msacc_gc_seconds_total" => "COUNTER",
+    "erlang_vm_msacc_other_seconds_total" => "COUNTER",
+    "erlang_vm_msacc_port_seconds_total" => "COUNTER",
+    "erlang_vm_msacc_sleep_seconds_total" => "COUNTER",
+    # "erlang_vm_msacc_alloc_seconds_total" => "COUNTER",
```

Comment:
I'm worried that these tests might become flakey.  Some metrics only show up in prometheus once they've actually registered a value.  The way these specs are currently written means that they are kinda dependent on what code has run before them.